src/test/java/manageezpz/logic/commands/DeleteEmployeeCommandTest.java

```diff
@@ -3,8 +3,8 @@
 import static manageezpz.logic.commands.CommandTestUtil.assertCommandFailure;
 import static manageezpz.logic.commands.CommandTestUtil.assertCommandSuccess;
 import static manageezpz.logic.commands.CommandTestUtil.showPersonAtIndex;
-import static manageezpz.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
-import static manageezpz.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static manageezpz.testutil.TypicalIndexes.INDEX_FIRST;
```

Comment:
Sorry ! @Denniszedead , would it be better to not change this in the first place?